docs/source/in5min.rst

```diff
@@ -0,0 +1,124 @@
+.. _in5min:
+
+PIConGPU in 5 Minutes
+=====================
+
+A guide to run, but not understand PIConGPU.
+It is aimed at existing and preconfigured high performance computing clusters (“supercomputers”, also: HPC systems).
+For more information on the individual steps please refer to the manual.
+
+This guide needs **shell access** (probably via ``ssh``) and **some auxilliary programs** (e.g. ``git``).
+Consider getting familiar with the shell and git.
+Please also read the tutorial for your local HPC cluster.
+
+We will use the following directories:
+
+- ``~/src/picongpu``: source files from github
+- ``~/picongpu.profile``: load the dependencies for your local environment
+- ``~/picongpu-projects``: scenarios to simulate
+- ``/scratch/mydir``: result data of the simulation runs
+
+Please replace them whenever appropriate.
+
+Get the source
+--------------
+
+Use git to obtain the source and use the current ``dev`` branch and put it into ``~/src/picongpu``::
+
+  mkdir -p ~/src
+  git clone -b dev https://github.com/ComputationalRadiationPhysics/picongpu ~/src/picongpu
+
+Setup
+-----
+
+You need :ref:`a lot of dependencies <install-dependencies>`.
+
+If you are on a known cluster you can take a shortcut:
+Select a profile file (``NAME.profile.example``) for your local HPC cluster from ``etc/picongpu`` and copy it to your home directory::
+
+  cp ~/src/picongpu/etc/picongpu/hemera-hzdr/k80_picongpu.profile.example ~/picongpu.profile
+
+This profile determines which part of the HPC cluster (“partition”, also: “queue”) – and thereby the type of GPUs – you will use.
```

Comment:
While we mainly use GPUs, there also CPU partitions and setups we support via alpaka. Thus perhaps using *hardware (e.g. GPU and CPU)* instead of the specific *GPU* would be better here.

Reply:
Or *device*

Reply:
addressed in a204c5003

I didn't use "device" because it is too much of a technical term (and even though in this context it typically refers to a device used for computation, it could also refer to a network card, storage device etc.) and I couldn't come up with a good, short explanation for it.

Reply:
Well we usually say *compute device*, don't we?

Reply:
\>_< bc7b5f438